ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -135,27 +135,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-baselinecaps-none
-- as: aws-ipi-byo-iam-role-p2-f28
```

Comment:
This one is being tracking in https://github.com/openshift/release/pull/40125/files#diff-692adf978c0074f0d0d11404a3887cb5976f96e3388d785d54393cc50fc1b49dL138.